app/configs/fatalitiesListViewColumns.tsx

```diff
@@ -25,34 +25,54 @@ export const fatalitiesListViewColumns: ColDataCardDef<fatalitiesListRow>[] = [
     sortable: true,
   },
   {
-    path: "law_enforcement_ytd_fatality_num",
-    label: "Law Enforcement Number",
+    path: "ytd_fatal_crash",
+    label: "YTD Fatal Crash",
     sortable: true,
   },
   {
-    path: "ytd_fatal_crash",
-    label: "YTD Fatal Crashes",
+    path: "ytd_fatality",
+    label: "YTD Fatality",
     sortable: true,
   },
   {
-    path: "ytd_fatality",
-    label: "YTD Fatalities",
+    path: "law_enforcement_ytd_fatality_num",
+    label: "LE #",
```

Comment:
trying to shorten these column headers and re-order them. will get feedback from Mary on this.